infra/horreum-infra-common/src/main/java/io/hyperfoil/tools/horreum/infra/common/resources/KeycloakResource.java

```diff
@@ -100,7 +97,7 @@ public void init(Map<String, String> initArgs) {
                         .withEnv("KC_DB_URL_HOST", "")
                         .withEnv("KC_HOSTNAME_STRICT", "false")
                         .withEnv("KC_HOSTNAME", "localhost")
-                        .withEnv("KC_DB_URL", "jdbc:postgresql://172.17.0.1:" + DB_PORT + "/keycloak")
+                        .withEnv("KC_DB_URL", "jdbc:postgresql://" + initArgs.get(HORREUM_DEV_POSTGRES_NETWORK_ALIAS) + ":5432/keycloak")
```

Comment:
Can we leave the `DB_PORT` as configurable? pple may spin up multiple instances on horreum in dev mode on the default network

Reply:
This is  going to break my workflow where I have multiple instances running in dev mode